samples/matter/light_switch/README.rst

```diff
@@ -422,12 +442,14 @@ Complete the following steps:
 Commissioning the device
 ========================
 
-.. include:: ../light_bulb/README.rst
-    :start-after: matter_light_bulb_sample_commissioning_start
-    :end-before: matter_light_bulb_sample_commissioning_end
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_commissioning_start
+    :end-before: matter_door_lock_sample_commissioning_end
 
 Before starting the commissioning procedure, the device must be made discoverable over Bluetooth LE.
-By default, the device is not discoverable automatically upon startup and **Button 4** must be used to enable the Bluetooth LE advertising.
+By default, the device is not discoverable automatically upon startup and the proper button must be used to enable the Bluetooth LE advertising:
```

Comment:
Actually, isn't it an oversight? We should probably fix it to start advertising automatically. Same for window-app.

Reply:
I remember that some time ago we wanted to trigger light switch advertising on button, as light bulb advertising starts again automatically. If we want to test both devices together we cannot control which device we will pair with controller without changing the default discriminator.

Reply:
But for window-app I agree, we should do that.

Reply:
oh, right.